<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Radio</class>
 <widget class="QMainWindow" name="Radio">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>145</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>541</width>
    <height>145</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>541</width>
    <height>145</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Steven Alspach</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>517</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="VolumeDial" name="volume_dial">
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="RadioTuner" name="tuner">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="maximum">
          <number>399</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="StationButton" name="preset_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StationButton" name="preset_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StationButton" name="preset_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StationButton" name="preset_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StationButton" name="preset_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDial" name="tuner_dial">
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StationButton</class>
   <extends>QPushButton</extends>
   <header>stationbutton.h</header>
  </customwidget>
  <customwidget>
   <class>VolumeDial</class>
   <extends>QDial</extends>
   <header>volumedial.h</header>
   <slots>
    <signal>switchedOn()</signal>
    <signal>switchedOff()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>RadioTuner</class>
   <extends>QSlider</extends>
   <header location="global">radiotuner.h</header>
   <slots>
    <slot>onChangeTuner(int)</slot>
    <slot>powerOn()</slot>
    <slot>powerOff()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>tuner_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tuner</receiver>
   <slot>onChangeTuner(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volume_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>volume_dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volume_dial</sender>
   <signal>switchedOn()</signal>
   <receiver>tuner</receiver>
   <slot>powerOn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volume_dial</sender>
   <signal>switchedOff()</signal>
   <receiver>tuner</receiver>
   <slot>powerOff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tuner(int)</slot>
 </slots>
</ui>
